pom.xml

```diff
@@ -1326,9 +1330,7 @@
                 </goals>
                 <phase>initialize</phase>
                 <configuration>
-                  <!-- ensure only one concurrent 'yarn install' -->
-                  <!-- when yarn cache is empty, multiple yarns performing network fetches frequently results in opaque errors -->
-                  <arguments>--mutex network</arguments>
+                  <arguments>${yarn.args}</arguments>
```

Comment:
`frontend-maven-plugin` already uses the value of the `frontend.yarn.arguments` property, if defined, so for consistency I suggest using that same property name rather than `yarn.args`. That would allow you to delete this `<arguments>` line (and therefore its parent `<configuration>`).

Reply:
(see https://github.com/eirslett/frontend-maven-plugin/blob/ec9f8050bc4c4014bbeb1ea84ae92e11682d9764/frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/YarnMojo.java#L26)

Reply:
Ok @basil , the only thing is that we do

```
 <properties>
    <yarn.version>1.22.19</yarn.version>
  </properties>
```

So you are suggesting that would now become

```
 <properties>
    <yarn.version>1.22.19</yarn.version>
    <frontend.yarn.arguments>1.22.19</frontend.yarn.arguments>
  </properties>
```

I am fine with it, but personally would find it more readable, if we would drop the `frontend` prefix since it just forces us to remember it. Better like:

```
 <properties>
    <yarn.version>1.22.19</yarn.version>
    <yarn.arguments>1.22.19</yarn.arguments>
  </properties>
```
wdyt?